apps/eec/fields/index.js

```diff
@@ -124,6 +135,26 @@ module.exports = {
       value: 'problem-valid-until'
     }
   },
+  'detail-signin-email': {
+    mixin: 'input-text',
+    validate: [
+      'email',
+      { type: 'maxlength', arguments: 254 }
+    ],
+    className: ['govuk-input', 'govuk-!-width-two-thirds'],
+    dependent: {
+      field: 'problem',
+      value: 'problem-signin-email'
+    }
+  },
+  'detail-signin-phone': {
```

Comment:
we can probably apply standard set of validators for phone number

```
if(phoneNumber.length > 0) {
        const phoneNumberWithoutSpace = phoneNumber.replace(/\s+/g, '').trim();
        const isValidphoneNumber = validators.regex(phoneNumberWithoutSpace, /^\(?\+?[\d()-]{8,16}$/);
        if(!isValidphoneNumber  || !validators.internationalPhoneNumber(phoneNumber)) {
          return validationErrorFunc('internationalPhoneNumber');
        }
      }
```

Reply:
should this field be `required`?

Reply:
Currently they've asked for these fields not to be validated, and not to be required even if they are selected as options

Reply:
Yes, all conditionally revealed fields in this page should now be required. `validations.json` updated as so.

Added HOF standard international/mobile validator